### PR TITLE
QRCode Detection Model

### PR DESCRIPTION
**Pull request by ermanok**

### CODE REVIEW

File: requirements.txt
Comment (rotx-maxim) [inline requirements.txt:1]: Can we fix qrcode to ==7.4.2

File: utils/WBAugmenter/WBEmulator.py
Comment (rotx-maxim) [inline utils/WBAugmenter/WBEmulator.py:1]: We need to add the LICENSE file text to this file

File: utils/WBAugmenter/imresize.py
Comment (rotx-maxim) [inline utils/WBAugmenter/imresize.py:1]: Fix 20242024

**Review by rotx-maxim — CHANGES_REQUESTED** [review]

File: requirements.txt
Comment (ermanok) [reply requirements.txt:1]: Sure

File: utils/WBAugmenter/imresize.py
Comment (ermanok) [reply utils/WBAugmenter/imresize.py:1]: fixed

**Review by rotx-maxim — APPROVED** [review]

**Review by seldauyanik-maxim — APPROVED** [review]

**Pull request merged**